### PR TITLE
In readme cap recommendation to current major release

### PR DESCRIPTION
**Pull request by manics**
This will allow us to make breaking changes without breaking existing Ansible role tests.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -21,7 +21,7 @@ Example `.travis.yml` file for testing Ansible roles:
       - docker
 
     install:
-      - pip install ome-ansible-molecule
+      - pip install "ome-ansible-molecule<0.5"
```
Comment (sbesson) [inline README.rst:24]: Makes me wonder whether we should have released it as `ome-ansible-molecule 1.0.0` and really capped on major releases

Comment (manics) [reply README.rst:24]: I'd support this proposal

**Comment by sbesson** [comment]
Merging for now and we can review the migration to `1.0.0` although this meta package is primarily used internally. Worth cutting a release @manics?

**Comment by manics** [comment]
Ansible 2.6.7 is out https://github.com/ansible/ansible/releases
How about hold off releasing until we bump to that?

**Pull request merged**